packages/cli/src/commands/help.ts

```diff
@@ -270,7 +270,7 @@ export function buildCommandExampleLines(command: Command) {
         outputArray.push(buildValueLine(line));
       }
     } else {
-      outputArray.push(buildValueLine(example.value));
+      outputArray.push(buildValueLine(example.value as string));
```

Comment:
I assume that's because the `const` makes those values into literals, so they have to be turned back into strings.

Reply:
@erikareads is on the right track. But for some reason, `Array.isArray()` was not correctly narrowing the `readonly` array type (from `as const`), so TS thinks this line could still be an array type (it can not).

Reply:
Can we add a comment above this line explaining that?

Reply:
Added comment in a follow-up PR https://github.com/vercel/vercel/pull/11955